sdk/support/Microsoft.Azure.Management.Support/src/Microsoft.Azure.Management.Support.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Support</PackageId>
+    <Description>Provides developers with libraries for the creation and management of SupportTicket resource.</Description>
```

Comment:
This is the description that will appear in Nuget - you should include a couple of sentences about the RP (allows users to manage Azure Support tickets).

Reply:
Updated description.